src/javascript/FormDefinitions/FormData.adapter.js

```diff
@@ -17,3 +19,29 @@ export const adaptSections = sections => {
         }, [])
         .filter(section => (section.fieldSets && section.fieldSets.length > 0));
 };
+
+export const getFieldValuesFromDefaultValues = field => {
```

Comment:
no test on this function ?

Reply:
it's already tested by higher call on other fct like adaptEditData or adaptCreateData, spent already a lot of time on tests, so I didnt want to extract the logics of this tests to test this extracted function